src/reason_pprint_ast.ml

```diff
@@ -857,17 +857,18 @@ let default = new Pprintast.printer ()
 type funcReturnStyle =
   | ReturnValOnSameLine
 
-let rec detectJSXComponent e attributes l =
-  match (e, attributes) with
+let rec detectJSXComponent e attributes l = None
```

Comment:
There's a note where this `detectJSXComponent` is called:

```
        (* This case will happen when there is something like

             Bar.createElement a::1 b::2 [] [@bla] [@JSX]

           At this point the bla will be stripped (because it's a visible
           attribute) but the JSX will still be there.
         *)
```

So it could just be that our unit tests aren't testing it?